src/clients/cassandra/CassandraConnector.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Promise = require('promise');
+
+module.exports = {
+  executeBatchMutations: executeBatchMutations,
```

Comment:
I'm a bit surprised that this export works given that the functions are declared as variables and therefore shouldn't be hoisted.